lib/views/home/home_header.dart

```diff
@@ -30,20 +34,24 @@ class HomeHeader extends StatelessWidget {
         image: profileImageUrl,
       );
 
-  Widget _profilePictureWidget(BuildContext context) => GestureDetector(
-        onTap: () {
-          // TODO: - Show the side bar to log out
-          // cannot pop as root is Home screen
+  Widget _profilePictureWidget(BuildContext context) => Consumer(
+        builder: (_, ref, __) {
+          return GestureDetector(
+            onTap: () {
+              ref.read(homeViewModelProvider.notifier).logOut();
+              context.go(RoutePath.login.path);
```

Comment:
I think updating to use `goNamed` or `pushNamed` is much better since we don't need to depend on the path or nested path anymore 😁

Reply:
You're right 👍